domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/gtfs/fareattributes/Transfers.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2020. MobilityData IO.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.domain.entity.gtfs.fareattributes;
+
+import java.util.stream.Stream;
+
+public enum Transfers {
+    NO_TRANSFERS_ALLOWED(0),
+    ONE_TRANSFER_ALLOWED(1),
+    UNLIMITED_TRANSFERS(2);
```

Comment:
Looking at http://gtfs.org/reference/static#fare_attributestxt:
>0 - No transfers permitted on this fare.
1 - Riders may transfer once.
2 - Riders may transfer twice.
empty - Unlimited transfers are permitted.

...`2` should be twice, and empty should be unlimited.